build.gradle.kts

```diff
@@ -19,9 +21,9 @@ plugins {
     alias(libs.plugins.kotlinx.binaryCompatibilityValidator)
 }
 
-apply plugin: 'kotlinx.team.infra'
+apply(plugin = "kotlinx.team.infra")
 
-infra {
+extensions.configure<InfraExtension> {
     teamcity {
         libraryStagingRepoDescription = project.name
```

Comment:
I would suggest rewriting this deprecation, as it seems to be a very easy change in the publishing block:
```
        sonatype {
            libraryStagingRepoDescription = project.name
        }
```

Reply:
I would appreciate the config update. Indeed, it's safer to do it in a separate commit.